core/src/main/java/com/sequenceiq/distrox/v1/distrox/StackOperations.java

```diff
@@ -315,6 +316,26 @@ public Stack getStackByCrn(String crn) {
         return stackCommonService.getRetryableFlows(name, workspaceId);
     }
 
+    public FlowIdentifier backupClusterDatabase(@NotNull NameOrCrn nameOrCrn, Long workspaceId, String location, String backupId) {
+        LOGGER.debug("Starting cluster database backup: " + nameOrCrn);
+        if (nameOrCrn.hasName()) {
```

Comment:
why can't this work by crn?

Reply:
I see, but if you check everything else (except upgrade, which I don't know why) supports both crn and name. I think this should too